src/views/latestBlocks/index.tsx

```diff
@@ -0,0 +1,106 @@
+import { formatDistanceToNowStrict } from 'date-fns';
+
+import { Icon } from '@components/icon';
+import { PDLink } from '@components/pdLink';
+import { PDScrollArea } from '@components/pdScrollArea';
+import { SearchBar } from '@components/searchBar';
+import { useStoreChain } from '@stores';
+import { cn } from '@utils/helpers';
+
+import styles from './styles.module.css';
+
+const LatestBlocks = () => {
+  const latestFinalizedBlock = useStoreChain.use.finalizedBlock?.();
+  const blocksData = useStoreChain?.use?.blocksData?.();
+
+  const formatString = (hash: string) => {
+    if (hash.length <= 8) {
+      return hash;
+    }
+    return `${hash.slice(0, 6)}...${hash.slice(-6)}`;
+  };
+
+  return (
+    <>
+      <div className="flex items-center">
+        <PDLink
+          to="/explorer"
+          className={cn(
+            'mr-8 duration-300 ease-out',
+            'bg-dev-purple-700 p-2 dark:bg-dev-purple-50',
+            'hover:bg-dev-purple-900 hover:dark:bg-dev-purple-200',
+          )}
+        >
+          <Icon
+            name="icon-arrowLeft"
+            className=" text-dev-white-200 dark:text-dev-purple-700"
+          />
+        </PDLink>
+        <h4 className="mr-2 font-h4-light">Latest Blocks</h4>
+      </div>
+      <SearchBar
+        label="Search by Block"
+        classNames="mt-6"
+      />
+      <PDScrollArea
+        className={styles['table-container']}
+        viewportClassNames="py-3"
+        verticalScrollClassNames="py-3"
+      >
+        <table className={styles['latest-blocks-table']}>
+          <thead>
+            <tr>
+              <th>Block</th>
+              <th>Status</th>
+              <th>Time</th>
+              <th>Extrinsics</th>
+              <th>Events</th>
+              <th>Validator</th>
+              <th>Block Hash</th>
+            </tr>
+          </thead>
+          <tbody>
+            {Array.from(blocksData.values()).reverse().map((block) => {
+              const timeAgo = block.header.timestamp && formatDistanceToNowStrict(
+                new Date(block.header.timestamp),
+                { addSuffix: true },
+              );
+
+              return (
+                <tr key={block.header.number}>
```

Comment:
We can talk with the UI/UX team if they want something like:
```
className={cn(
  {
    ['opacity-0 animate-fade-in animation-duration-500 animation-delay-500']: blockIndex === 0,
  },
)}
```
for better visualization of the new block

Reply:
Done